### PR TITLE
added pytest-azurepipelines

### PR DESCRIPTION
**Pull request by nabobalis**
Lets see what it does.

### CODE REVIEW

File: azure-templates.yml [hunk 1]
```diff
@@ -42,25 +42,11 @@ jobs:
     - script: pip install --upgrade tox-conda
       displayName: install tox-conda
 
-  - script: tox -e ${{ parameters.tox }} --junitxml=junit/test-results.xml --cov-report=xml --cov-report=html
+  - script: tox -e ${{ parameters.tox }} --junitxml=junit/test-results.xml --cov-report=html
```
Comment (Cadair) [inline azure-templates.yml]: We can drop the junit as well?

**Comment by Cadair** [comment]
we appear to have broken codecov?

**Comment by nabobalis** [comment]
Did it?

**Comment by nabobalis** [comment]
It really did break it.

**Pull request closed without merging**